src/Process/Listener/TextFileLoggerSubscriber.php

```diff
@@ -76,12 +81,15 @@ private function getLogParts(array $processes, string $headlinePrefix): array
         $logParts = [sprintf('%s mutants:', $headlinePrefix), ''];
 
         foreach ($processes as $index => $mutantProcess) {
+            $mutant = $mutantProcess->getMutant();
```

Comment:
Just before your PR, I pushed a new branch with similar changes, but (from my point of view of course) with a bit prettier output

Please see https://github.com/infection/infection/compare/better-log?expand=1

Example of output for your repo:

```bash
Not covered mutants:
====================


1) Infection\Mutator\FunctionSignature\PublicVisibility
/Users/user/tmp/rest-bundle/src/Request/Modifier/RequestModifierInterface.php:19

2) Infection\Mutator\FunctionSignature\PublicVisibility
/Users/user/tmp/rest-bundle/src/Response/Modifier/ResponseModifierInterface.php:21

3) Infection\Mutator\Boolean\TrueValue
/Users/user/tmp/rest-bundle/src/Response/ResponseRenderer.php:71

4) Infection\Mutator\FunctionSignature\PublicVisibility
/Users/user/tmp/rest-bundle/src/Transformer/TransformerInterface.php:21

5) Infection\Mutator\FunctionSignature\PublicVisibility
/Users/user/tmp/rest-bundle/src/Transformer/TransformerInterface.php:28
```

It's better because

1. it has `file:lineNum` notation which perfectly works with `ctrl+click` and opens exact line in your editor
2. it does not have diff for uncovered mutation, this *significantly* reduces the log
3. it has two separate methods where not needed information is removed for uncovered tests (e.g. we don't need command line for it, we don't need process output (even `''`))

Besides that, there is some changes to make ouput more readable.

I suggest to use code from that branch, or merge it to your branch somehow

Reply:
Thanks for reply!
I agree with 1 and 3 and i will try to merge your changes.
In my opinion the log should be detailed as possible. I have an example, but I can show it in the evening.

Reply:
There is an example of the last uncovered mutation:
```diff
1) Infection\Mutator\Boolean\TrueValue
/mmedia/projects/oxid-rest-bundle/src/Response/ResponseRenderer.php
exec /usr/bin/php7.1 /mmedia/projects/oxid-rest-bundle/vendor/phpunit/phpunit/phpunit --configuration /tmp/infection/phpunitConfiguration.1454e421d53db78e46101421c0968c75.infection.xml --stop-on-failure
--- Original
+++ New
@@ @@
     {
-        switch (true) {
+        switch (false) {
             case $data === null:
                 return new NullResource();
             case is_array($data):
             case $data instanceof \Traversable:
                 return new Collection($data, $this->transformer, $resourceKey);
             default:
```

Code coverage still 100%. It looks like false positive and with logs i can check it.

Reply:
I've created an issue in PHPUnit_Coverage repository. Please read/track it here: https://github.com/infection/infection/issues/34